src/components/AddActivityList.stories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import AddActivityForm from './AddActivityForm'
+import GlobalStyle from '../GlobalStyle'
```

Comment:
Hast du die GlobalStyles zufällig in eine storybook preview.js Datei ausgelagert? ;) Hier benutzt du die GlobalStyles und den Decorator gar nicht. Das macht dann deine imports etwas schlanker :)

Reply:
Nein, werde ich mich mal etwas ausgiebiger mit beschäftigen. Danke dir für den Hinweis! :-)